modules/custom/falcon_thankq/falcon_thankq.info.yml

```diff
@@ -0,0 +1,5 @@
+name: Falcon ThankQ
+type: module
+description: ThankQ integration and modifications for Falcon.
+core: 8.x
+package: Falcon
```

Comment:
Shouldn't we have dependency on `inline_entity_form` here?

Reply:
@duozersk great catch, indeed.